bash-preexec.sh

```diff
@@ -47,7 +47,17 @@ __bp_last_argument_prev_command="$_"
 
 # Command to set our preexec trap. It's invoked once via
 # PROMPT_COMMAND and then removed.
-__bp_trap_install_string="trap '__bp_preexec_invoke_exec \"\$_\"' DEBUG;"
+__bp_trap_install() {
+  local prior_trap=$(trap -p DEBUG | sed "s/[^']*'\(.*\)'[^']*/\1/")
+  if [[ -n "$prior_trap" ]]; then
+    eval '__bp_original_debug_trap() {
+      '"$prior_trap"'
+    }'
+    preexec_functions+=("$__bp_original_debug_trap")
```

Comment:
Oof, good catch. I realize my test is also missing the point, since the `DEBUG` trap is invoked immediately. I've got a better test pending.

This would also have been caught if we didn't [silently skip missing functions](https://github.com/rcaloras/bash-preexec/blob/b6a9b9a/bash-preexec.sh#L98) - I'd been thinking about suggesting changing that, I'll file a bug unless you'd prefer to leave it be.